mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule Protobuf.Mixfile do
   use Mix.Project
 
   @source_url "https://github.com/elixir-protobuf/protobuf"
-  @version "0.12.0"
+  @version "0.13.0"
```

Comment:
Can you not bump the version quite yet? We want to release multiple things at once, so let's leave this as it's in `main`.

Reply:
Reverted 👍